### ISSUE TITLE
[PLW003v2-IP03] | Illegile “allow” message

### ISSUE THREAD

**Issue by function0xMarki** [syscoin/pali-wallet#716]
### What happened?

When importing a token, Pali Wallet displays a warning message labeled “Allow” to prompt user confirmation. However, the design of this message presents a visual accessibility issue: the text appears in black over a dark blue background, making it nearly unreadable. This lack of contrast hinders message comprehension, as it can only be read if the user is already aware of its presence and manually highlights the text to reveal it.

![Image](https://github.com/user-attachments/assets/544324fc-9e9a-4fa9-b3d7-431c66e6fe92)

### Version

v.3.5.0

### What platform are you using?

macOS

### What browser are you using?

Brave

### Is this a trezor issue?

No

### Relevant log output

```shell

```

**Comment by sidhujag** [comment]
was this done via dapp (it shows external)

**Comment by function0xMarki** [comment]
> was this done via dapp? (it shows external)

It is when trying to make the web3 connection of Pali Wallet v3.5.0 with the "BlockScout" explorer of Rollux.

**Comment by will07dev** [comment]
### **What happened?**
During testing I was unable to trigger the expected “Allow” permission prompt, so the message never appeared.

**Comment by will07dev** [comment]
### **What happened?**
During testing I was unable to trigger the expected “Allow” permission prompt, so the message never appeared.
### **Version**
v.3.5.0-0711

### **What platform are you using?**
Windows 11

### **What browser are you using?**
Brave

### **Is this a trezor issue?**
No

**Comment by will07dev** [comment]
### **What happened?**
During testing I was unable to trigger the expected “Allow” permission prompt, so the message never appeared.

### **Version**
v.3.5.0-0716

### **What platform are you using?**
Windows 11

### **What browser are you using?**
Brave

### **Is this a trezor issue?**
No

**Comment by sidhujag** [comment]
try again in latest

**Comment by will07dev** [comment]
### **What happened?**
During testing I was unable to trigger the expected “Allow” permission prompt, so the message never appeared.

### **Version**
v3.5.0-0727

### **What platform are you using?**
Windows 11

### **What browser are you using?**
Brave

### **Is this a trezor issue?**
No

**Comment by sidhujag** [comment]
allow as in request connection? or connect account? two seperate screens. There are middlewares that run every request and check if it needs to have auth/connection/network type utxo vs evm for switching etc. Now the token watch asset page should just show token details to add, there is no such message in watching asset.

**Comment by function0xMarki** [comment]
✅ Solved in v4.0.3

<img width="381" height="179" alt="Image" src="https://github.com/user-attachments/assets/c9a6d9ef-9ad0-4474-87c7-8b99e0bf2871" />
